metrotranscom/doorway
# Release/2024 03 20

# Pull Request Template

## Issue Overview

This PR addresses [3818](https://github.com/bloom-housing/bloom/issues/3818)


## Description

Pulls the full passwordless feature set from core + also Morgan's geocoding fix.

## How Can This Be Tested/Reviewed?

Locally toggle on the public site feature toggle, and you can setup the backend with either dev or staging.
The full passwordless features should work, including creating an account and signing in with a code.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
